source/JABHandler.py

```diff
@@ -836,10 +840,15 @@ def enableBridge():
 
 
 def initialize():
-	global bridgeDll, isRunning
+	global bridgeDll, is_64Bit, isRunning
+	# If the system is 64-bit, load the dll that we have in the NVDA distribution.
+	# Otherwise, it loads the one on the 32-bit system, which does not have the -32 suffix.

```

Comment:
How can we guarantee this is installed? Shouldn't we be distributing this ourselves?

Reply:
are you certain we need this other DLL? The one we use is meant to work for 32bit systems. We are specifically using a 32bit DLL.

Reply:
Yes. First for seeing the code before the change to Python 3, for doing the tests and for having read the jdk code.
He explains that WindowsAccessBridge-32.dll is for accessing 64-bit applications using a 32-bit environment. However, only WindowsAccessBridge.dll is for the 32-bit operating system.

Reply:
As I explained in the mentioned issue, it would be good to include the dll.
Since it is obtained by compiling JDK code, it is not in my scope. On my local machine I did this, even the code was programmed to look for it in a specific path.
I don't think anyone runs Java 6 or lower.
If we include the dll I think it is better. If Java is updated it will not be blocked by NVDA.

Reply:
Please follow the instructions in https://github.com/nvaccess/javaAccessBridge32-bin and create a PR updating JAB and including this dll

The Java build we get from azul contains the required `javaaccessbridge.dll`

Reply:
Now what's left to do is compile the other dll, without the -32 suffix. I have modified the code to point to the same path of the current dll.

Reply:
you don't need to compile it - you can follow the instructions in the repository to download a compiled version